src/pymodaq_gui/utils/utils.py

```diff
@@ -143,6 +148,8 @@ def h5tree_to_QTree(base_node, base_tree_elt=None, pixmap_items=[]):
         base_tree_elt = QtWidgets.QTreeWidgetItem([base_node.name, "", base_node.path])
     for node_name, node in base_node.children().items():
         child = QtWidgets.QTreeWidgetItem([node_name, "", node.path])
+        if hasattr(node, 'title') and node.title:
```

Comment:
could you add that is the 'origin' is an attribute it will build the tooltip as : f'{origin}/{title}'